README.md

```diff
@@ -314,7 +314,6 @@ class RefundResponse:
     refund_status: RefundStatus
     order_id: str  # the ID of the order that needs refunding
     failure_reason: Optional[str] = None  # Optional, only populated on failure
-    receipt_json: Optional[str] = None  # Optional, json blob for receipt data
```

Comment:
isn't it possible to issue a receipt, the same way that we do on the terminals?

Reply:
We did, but the information that we got were not refund related, so after a discussion with Simon we removed it

Reply:
In that case we need to document how can they print a refund receipt. Which fields to use to from the original payment receipt_json, and what other information (for example the refund id if exists one)?